sdk/tests/tutorials/ibor/test_properties.py

```diff
@@ -19,7 +19,7 @@ class Properties(unittest.TestCase):
     @classmethod
     def setUpClass(cls):
         # create a configured API client
-        api_client = ApiClientBuilder().build(CredentialsSource.secrets_path())
+        api_client = ApiClientBuilder().build()
```

Comment:
was a need to do this? it please that locally devs will now need to use env vars rather that a secrets file

Reply:
OK, there were a couple of them.  I'll need to rethink how to do that then.  If you have the env vars, you want them to trump the secrets file if it even exists.

Reply:
sorted.